### ISSUE TITLE
libica 3.3.x has executable stack

### ISSUE THREAD

**Issue by sharkcz** [opencryptoki/libica#20]
Seems libica 3.3.x binary now contains executable stack, which is a security issue. I suppose it comes with the generated assembly code in mp.S

````
[sharkcz@devel11 src]$ execstack /usr/lib64/libica.so.3.3.3
X /usr/lib64/libica.so.3.3.3
````

**Comment by p-steuer** [comment]
bisect shows it was indeed introduced with the assembly code of b7bc2baa61cc212f43efa1bf81010d108609d1c4 and went unnoticed.

Thanks for reporting.

Should be fixed with PR #21 .

**Comment by p-steuer** [comment]
Fixed with PR #21
